### ISSUE TITLE
Accessibility: apply wai-aria methods to alert screen reader users when multiple options have been selected

### ISSUE THREAD

**Issue by chamandeep** [silviomoreto/bootstrap-select#541]
operating system: Windows 7 64 bit
browser: all versions of IE and Firefox
screen reader: Jaws v15 and NVDA 2014.
select picker version: bootstrap 3.0.2

Actual result: When using the functionality to select multiple options, there is currently not an easy way to identify whether an option has been selected. As a blind user using a screen reader, i believe there is a visual icon to highlight a selected choice, but this is not accessible.
expected result: wai-aria or alternative methods are used to determine when an option has been selected. The screen reader is able to tell that an option is selected by using a phrase such as "coke, selected'.

**Comment by petergraff** [comment]
Hi, 
With regard to this issue as well as [#1222](https://github.com/silviomoreto/bootstrap-select/issues/1222) and [#870](https://github.com/silviomoreto/bootstrap-select/issues/870), just wondering if there's a plan to incorporate overall consistent use of ARIA roles in the near future? Thanks.

**Comment by georgebatalinski** [comment]
Any update here? are you accepting PRs on this?

**Comment by caseyjhol** [comment]
todo: aria-selected attribute on selected options
